build/AzurePipelinesTemplates/WindowsAppSDK-RunHelixTests-Job.yml

```diff
@@ -134,6 +134,14 @@ jobs:
   #     filePath: $(winUIHelixPipelineScripts)\GenerateHelixWorkItems.ps1
   #     arguments: -TestFilePattern 'mrtcore_test_binaries_x64\MrtCoreManagedTest\*.dll' -TestBinaryDirectoryPath '$(testPayloadDir)' -OutputProjFile '$(helixWorkItemsDir)\MrtCore-MrtCoreManagedTest.proj' -WorkItemPrefix 'MrtCore-MrtCoreManagedTest-$(buildPlatform)-$(buildConfiguration)' -TestNamePrefix 'MrtCore.MrtCoreManagedTest.$(buildPlatform).$(buildConfiguration)'
 
+  # Discover the TAEF Push Notifications test binaries to run and generate the Helix Work Items for them.
+  - task: powershell@2
+    displayName: 'Discover Push Notifications UnitTests'
+    inputs:
+      targetType: filePath
+      filePath: $(winUIHelixPipelineScripts)\GenerateHelixWorkItems.ps1
+      arguments: -TestFilePattern '$(Build.SourcesDirectory)\BuildOutput\$(buildConfiguration)\$(buildPlatform)\PushNotificationTests\*.dll' -TestBinaryDirectoryPath '$(testPayloadDir)' -OutputProjFile '$(helixWorkItemsDir)\PushNotificationTests.proj' -WorkItemPrefix 'MrtCore.MrmUnitTests.$(buildPlatform).$(buildConfiguration)' -TestNamePrefix 'PushNotificationTests.$(buildPlatform).$(buildConfiguration)'
+
```

Comment:
@jefgen have those aggregator pipeline test changes been replicated here yet? (I haven't done it)

And how about that data-driven approach we discussed? Waaay preferable to developers editing the pipeline for new tests, and with all the myriad fine print we discussed...

Reply:
@DrusTheAxe , @jefgen, That's an old side projects (adding the push notifications unit tests to the build pipeline) that I've just reactivated for EIM. I've got about two weeks left before going on vacation that I can work on this. If you care to brief me on the data -driven approach (or other plans) maybe I can be of some help.